src/harness/certs/generate_fake_certs.sh

```diff
@@ -7,8 +7,11 @@ sed -i '/TEST = critical, ASN1:NULL/d' ../../../cert/openssl.cnf
 rm -rf crl/
 mkdir private
 mkdir root
-touch index.txt
-echo -n 'unique_subject = no' >> index.txt.attr
+mkdir crl
+
+# Empty the index.txt to avoid to old certificates index.
+cat /dev/null > index.txt
```

Comment:
Couldn't you just use `rm`?

Reply:
sub CA's   are named correctly to give better clarity